packages/ra-ui-materialui/src/input/ReferenceInput.js

```diff
@@ -54,6 +54,7 @@ export const ReferenceInputView = ({
     classes,
     className,
     error,
+    fullWidth,
```

Comment:
This is exactly the type of props that we want to pass with `rest` instead of identifying them individually here.

Reply:
I did this, because of the `rest` props were still being passed to `Labeled`, this is no long the case so this can be changed.